src/Event.php

```diff
@@ -167,6 +167,11 @@ final class Event
      */
     private $type;
 
+    /**
+     * @var Attachment[] The attachments associated with this event
+     */
+    private $attachments = [];
```

Comment:
This is the main diversion from the Javascript SDK which I drew inspiration from. It puts this field into `EventHint` instead. Dunno if it makes more sense that way, but it would require passing the hint along with the event to the transport layer, and thus a huge BC break in the `TransportInterface`

Reply:
This is fine to me.